api/v1alpha1/opentelemetrycollector_types.go

```diff
@@ -51,6 +51,12 @@ type OpenTelemetryCollectorSpec struct {
 	// +operator-sdk:gen-csv:customresourcedefinitions.specDescriptors=true
 	ServiceAccount string `json:"serviceAccount,omitempty"`
 
+	// VolumeClaimTemplates will provide stable storage using PersistentVolumes.
```

Comment:
```suggestion
	// VolumeClaimTemplates will provide stable storage using PersistentVolumes. Only available when the mode=statefulset.
```

Reply:
Applied in latest commit: "Fix volumeclaim function and small bugs"